frontend/app/abtests/BundleVariant.scala

```diff
@@ -49,8 +48,8 @@ object BundleTier {
 
 
 object Distribution {
-  val A = Distribution("A")
-  val B = Distribution("B")
+  val CONTROL = Distribution("CONTROL")
```

Comment:
The test no longer refers to A1 or A2. Keeping those identifiers here adds confusion. Better to make the code on both sides consistent so someone unfamiliar has a chance to understand what's going on. Because we have to have a `control` group on the frontend side for Abacus, we're going to see it here too.